src/Sylius/Behat/Context/Api/Admin/ManagingProductAttributesContext.php

```diff
@@ -0,0 +1,430 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use Behat\Behat\Context\Context;
+use Ramsey\Uuid\Uuid;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\ResponseCheckerInterface;
+use Sylius\Behat\Context\Api\Resources;
+use Sylius\Behat\Service\SharedStorageInterface;
+use Sylius\Component\Product\Model\ProductAttributeInterface;
+use Webmozart\Assert\Assert;
+
+final class ManagingProductAttributesContext implements Context
+{
+    public function __construct(
+        private ApiClientInterface $client,
+        private ResponseCheckerInterface $responseChecker,
+        private SharedStorageInterface $sharedStorage,
+    ) {
+    }
+
+    /**
+     * @When I want to see all product attributes in store
+     */
+    public function iWantToBrowseProductAttributes(): void
+    {
+        $this->client->index(Resources::PRODUCT_ATTRIBUTES);
+    }
+
+    /**
+     * @When /^I(?:| try to) delete (this product attribute)$/
+     */
+    public function iDeleteThisProductAttribute(ProductAttributeInterface $attribute): void
+    {
+        $this->client->delete(Resources::PRODUCT_ATTRIBUTES, $attribute->getCode());
+    }
+
+    /**
+     * @When I want to create a new :type product attribute
+     */
+    public function iWantToCreateANewTypedProductAttribute(string $type): void
+    {
+        $this->client->buildCreateRequest(Resources::PRODUCT_ATTRIBUTES);
+        $this->client->addRequestData('type', $type);
+    }
+
+    /**
+     * @When I specify its code as :code
+     */
+    public function iSpecifyItsCodeAs(string $code): void
+    {
+        $this->client->addRequestData('code', $code);
+    }
+
+    /**
+     * @When I name it :name in :language
+     * @When I change its name to :name in :language
+     * @When I do not name it
+     * @When I remove its name from :language translation
+     */
+    public function iNameItIn(string $name = '', string $language = 'en_US'): void
+    {
+        $this->client->updateRequestData(['translations' => [$language => ['name' => $name, 'locale' => $language]]]);
+    }
+
+    /**
+     * @When I (also) add value :value in :language
```

Comment:
```suggestion
     * @When I (also) add value :value in :localeCode
```
This is more precise IMO